bin/main.rb

```diff
@@ -1,23 +1,56 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-puts "Welcome to Tic Tac Toe!"
+require_relative "../lib/board"
+require_relative "../lib/player"
+
+winning_combinations = [ [1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7] ]
+
+puts 'Welcome to Tic Tac Toe!'
 puts "It's a two player game, so we'll need your names."
-puts "Enter Player 1: "
-player1 = gets.chomp
-puts "Enter Player 2: "
-player2 = gets.chomp
-#create variables to hold "winning combinations", "Player 1's choices" and "Player 2's choices"
-puts ""
-#Prompt player for choices until a winning combination of choices or a tie is obtained
-print "###\n###\n###\n"
-puts ""
-puts "#{player1} your turn, choose a cell[1-9]: "
-puts "#{player1} choices are updated"
-puts ""
-print "###\n###\n###\n"
-puts ""
-puts "#{player2} your turn, choose a cell[1-9]: "
-puts "#{player2} choices are updated"
-puts ""
-puts "The Winner is: #{player1}"
+puts 'Enter Player 1: '
+playerX = gets.chomp
+player1 = Player.new(playerX, 'X')
+puts 'Enter Player 2: '
+playerO = gets.chomp
+player2 = Player.new(playerO, 'O')
```

Comment:
What happens if players enter an empty string as their name? Consider having a default name or tell the user to enter a valid name.

[OPTIONAL]
What if they enter the same name for both players? :wink: